### DISCUSSION TITLE
How to Check Facebook Advertisement Tracking is Successfully setup in Debug

### DISCUSSION THREAD

**Discussion by ThushalIntervest** [thebergamo/react-native-fbsdk-next#351] [Q&A]
How to check if my Facebook advertisement is Successfully initialised in debug mode?

Here I use this code to Track the advertisement. Yes I know I can see it in the Facebook developer dashboard. But I want to see the success or response. If someone knows the answer, Please Give me a solution for it.

`Settings.setAdvertiserTrackingEnabled(true);`

**Comment by mikehardy** [comment]
You may find more response from stackoverflow, perhaps even more likely if you narrow it to either android or iOS and ask for specifics for that platform (since we're a pretty thin wrapper over the platform-specific native SDKs any responses you receive should be useful still)
